### PR TITLE
solved #19 Added : splash screen

### PR DESCRIPTION
**Pull request by sameersrivastava13**

### CODE REVIEW

**Comment by abhu-A-J** [comment]
> _No description provided._
@sameersrivastava13 
Link issue and add a screenshot or gif showing work!

**Comment by anushbhatia** [comment]
Splash Screen gif looks kiddish and not relevant to medical project

**Comment by abhu-A-J** [comment]
> Splash Screen gif looks kiddish and not relevant to medical project

It seems he didn't read the comments on the issue itself! @sameersrivastava13 I had given a resource use that for loader inspiration as suggested! :+1:

File: assets/css/style.css [hunk 1]
```diff
@@ -8,6 +8,25 @@ body {
 	max-width: 1592px;
 	margin: 0 auto;
 }
+
+#loading{
```
Comment (anushbhatia) [inline assets/css/style.css:12]: Don't use the loader till it finishes loading
Instead have it till the complete page loads
[Use this](https://smallenvelop.com/display-loading-icon-page-loads-completely/)

**Comment by sameersrivastava13** [comment]
> Splash Screen gif looks kiddish and not relevant to medical pr

> Splash Screen gif looks kiddish and not relevant to medical project

is this one okay? or should i use some other.

splash

**Comment by anushbhatia** [comment]
try using shades of black and white

File: assets/css/style.css (hunk 1 above)
Comment (sameersrivastava13) [reply assets/css/style.css:12]: okay

**Comment by sameersrivastava13** [comment]
> > Splash Screen gif looks kiddish and not relevant to medical project
> 
> It seems he didn't read the comments on the issue itself! @sameersrivastava13 I had given a resource use that for loader inspiration as suggested! 👍

okay i'll try making it from scratch instead of using a GIF.

**Comment by sameersrivastava13** [comment]
> try using shades of black and white

Okay sure

**Comment by smaranjitghose** [comment]
Any updates?

**Comment by sameersrivastava13** [comment]
> Any updates?

Working on it.

**Comment by sameersrivastava13** [comment]
> > Any updates?
> 
> Working on it.

Done.
i have added this loader GIF.

splash

**Pull request merged**